nexus/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: default
```

Comment:
I think its a typo 
```suggestion
  name: nexus
```

Reply:
no, it's not typo. when performing it , the namespace will be replaced by the namesapce value in `kustomization.yaml`

Reply:
Ahhh I didn't notice that. Thank you